libmscore/textbase.cpp

```diff
@@ -519,8 +519,35 @@ void TextFragment::draw(QPainter* p, const TextBase* t) const
       {
       QFont f(font(t));
       f.setPointSizeF(f.pointSizeF() * MScore::pixelRatio);
+#if defined Q_OS_WIN || defined Q_OS_LINUX
+      if (f.bold()) {
+            // workaround for https://musescore.org/en/node/284218
+            // and https://musescore.org/en/node/281601
+            // only needed for certain artificially emboldened fonts
+            // see https://musescore.org/en/node/281601#comment-900261
+            // in Qt 5.12.x this workaround should be no more necessary if
+            // env variable QT_MAX_CACHED_GLYPH_SIZE is set to 1
```

Comment:
Would it make sense then to use `if (QT_VERSION >= QT_VERSION_CHECK(5, 12, 0))`?

Reply:
Maybe, but we should also check that it is possible to defined QT_MAX_CACHED_GLYPH_SIZE=1 inside MuseScore code (maybe before all the initializations) and that it works without slowing the program (in my pc it does not slow down the program, but better safe than sorry).
This comment is just a reminder that in the future (if and when we will use a newer version of Qt) the workaround could be no more needed.